diff --git a/node_modules/@clerk/shared/dist/react/index.mjs b/node_modules/@clerk/shared/dist/react/index.mjs
index 8c5c0e1..9f75dc3 100644
--- a/node_modules/@clerk/shared/dist/react/index.mjs
+++ b/node_modules/@clerk/shared/dist/react/index.mjs
@@ -56,15 +56,16 @@ var createContextAndHook = (displayName, options) => {
 import React2 from "react";
 
 // src/react/clerk-swr.ts
+import * as swr_star from "swr";
+import * as swr_infinite from "swr/infinite";
+
 var clerk_swr_exports = {};
 __export(clerk_swr_exports, {
-  useSWR: () => default2,
-  useSWRInfinite: () => default3
+  useSWR: () => swr_star.useSWR,
+  // useSWRInfinite: () => swr_infinite.useSWRInfinite
 });
 __reExport(clerk_swr_exports, swr_star);
-import * as swr_star from "swr";
-import { default as default2 } from "swr";
-import { default as default3 } from "swr/infinite";
+
 
 // src/react/contexts.tsx
 var [ClerkInstanceContext, useClerkInstanceContext] = createContextAndHook("ClerkInstanceContext");
